### PR TITLE
Add nbi and ec samples to omas

### PR DESCRIPTION
**Pull request by jmcclena**
@eldond

### CODE REVIEW

**Comment by orso82** [comment]
👍 @jmcclena can you please add the `@add_to_ODS` to the new sample functions

**Comment by orso82** [comment]
@jmcclena now that the nbi and ec are being regression tested some issues came up:
https://travis-ci.com/github/gafusion/omas/builds/173921455

looks like `pi --> numpy.pi`

**Comment by jmcclena** [comment]
> @jmcclena now that the nbi and ec are being regression tested some issues came up:
> https://travis-ci.com/github/gafusion/omas/builds/173921455
> 
> looks like `pi --> numpy.pi`

ahhh thanks for finding that. One day I'll get my own version of omas working on iris again so I can debug these things myself.

File: omas/omas_sample.py [hunk 1]
```diff
@@ -729,6 +733,7 @@ def nbi(ods, nunits=2, ntimes=6):
     times = numpy.linspace(0, 1, ntimes)
     ones = numpy.ones(ntimes)
 
+    ods['nbi.time'] = times
```
Comment (eldond) [inline omas/omas_sample.py:736]: @jmcclena `ods.plot_nbi_summary()` failed without this. This line made it work on my machine, but did I do it right?

**Pull request merged**